src/App.js

```diff
@@ -1,21 +1,33 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
+import React, {Component} from 'react';
+import Header from './components/header';
+import Search from './components/search';
+import Info from './components/info';
 import './App.css';
+import AuthService from './components/helper/auth';
+import withAuth from './components/config/constants';
+const Auth = new AuthService();
 
 class App extends Component {
+  handleLogout() {
+    Auth.logout()
+    this
+      .props
+      .history
+      .replace('/login');
+  }
+
   render() {
     return (
       <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
+        <button type="button" className="form-submit" onClick={this.handleLogout}>Logout</button>
```

Comment:
Are you sure this onClick event triggers the `handleLogout` method? you need to bind it .